unfccc/client.go

```diff
@@ -3,64 +3,43 @@ package unfccc
 import (
 	"context"
 	"encoding/json"
-	"fmt"
-	"os"
+	"io/ioutil"
```

Comment:
As of go 1.16 [os.ReadFile is preferred](https://pkg.go.dev/io/ioutil) but it causes CI to fail :(

ioutil will be around for a while so I switched to that.